### ISSUE TITLE
[Design] My VA - Payments Information: Update Frontend documentation

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#33993]
## Background

Update Frontend documentation, based on design and team discussions

## Tasks

- [x] Meet with Frontend (Taylor) to review [final design](https://app.mural.co/t/vsa8243/m/vsa8243/1640028268317/2ba4e1b2846c87f5d0bfc6bcb643bef84720bea1?sender=30b328dd-61fa-4e9c-a615-80a4db99566d), if needed.
- [x] Update [documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/payment-history/documentation/payment-history-frontend.md) to include up-to-date screenshots
- [x] Ensure documentation includes error states

## Acceptance Criteria

- [ ] Confirm design details and proposal of functionality is clear and agreed upon
- [ ] GH document reflects agreed upon design and functionality

**Comment by hdjustice** [comment]
As we have moved forward with prototype development and research sessions, the Frontend Documentation has been an ongoing update.  Now that research is complete, we have several design updates that will be needed, and will impact this documentation.
Moving to next sprint so that we can finalize the design and ensure this document reflects those changes.

**Comment by andaleliz** [comment]
@hdjustice I have a question for you - I notice the current documentation includes states regarding whether or not a user has new messages. This is because as part of this effort, we've also updated styling of the "unread messages" link to follow the new alert pattern. 

Is that part of the onsite notification work though? Or is it just it's own separate update that got mixed in? 

It feels distinct from the payment information section of the page. 

I don't want to confuse the two, or overly complicate the specs and states in the payment documentation if it is a separate thing.

**Comment by hdjustice** [comment]
@andaleliz I was thinking about this too.  The changing of the verbiage is separate from everything else, but got mixed in because of the styling change.  When I created the ticket initially to update the verbiage, I inadvertently included the styling as well so Taylor coded for it.. then we had to pull it back out because Samara was concerned that we might be changing the style.
I was thinking to release the change at the same time, to keep the styling consistent, however I can see your point in not overly complicating it.  I am going to ask Samara about it, since she wanted to wait on that.

**Comment by hdjustice** [comment]
@andaleliz - Documenting: Per this [Slack thread](https://dsva.slack.com/archives/C909ZG2BB/p1644243039170769), Samara would like to update the style for the unread messages at the same time for consistency.  I will be creating another ticket for the style change only, and ensure the change is kept with the same feature tag as payment info.  The verbiage will be updated separately and applied once coded.  It will not have a dependency on the payment info release.

**Comment by hdjustice** [comment]
FYI @andaleliz - I created a **[new folder](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/my-va/Health%20care%20-%20unread%20messages%20style%20change)** for the health care style change

**Comment by andaleliz** [comment]
Thanks @hdjustice! I have another question - there's some back and forth in the current doc about whether or not the payment made card is shown if they've received a payment in the **last** 30 days, or if they have a payment coming in the **next** 30 days.

The former is correct right? The card isn't based on scheduled payments?

**Comment by hdjustice** [comment]
That is correct.  We will only be listing past payments, not looking forward to upcoming payments.

**Comment by andaleliz** [comment]
@hdjustice thanks for your patience will all my questions along the way! I think [this documentation ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/payment-history/documentation/payment-history-frontend.md) is done. When you have a moment, would you mind reviewing the docs and make sure I didn't miss anything?

**Comment by hdjustice** [comment]
The document looks great!  I checked against the test cases and I think you have everything covered!

**Comment by hdjustice** [comment]
I will make sure the error states get added in there.  That should round out everything nicely

**Comment by andaleliz** [comment]
Thanks so much Heather! I missed that task in the description.

**Comment by andaleliz** [comment]
@hdjustice thank you so much - it looks great! I added a bullet under the 2nd and 3rd error state to clarify that the debt/payment info that _does not_ fail should appear under the alert component. 

Closing this out :)
